cmd/notation/cert/delete.go

```diff
@@ -33,6 +33,17 @@ func certDeleteCommand(opts *certDeleteOpts) *cobra.Command {
 			}
 			return nil
 		},
+		Long: `Delete certificates from the trust store
+
+Example - Delete all certificates with "ca" type from the trust store "acme-rockets":
+  notation certificate delete --type ca --store "acme-rockets" --all
+
+Example - Delete certificate "cert1.pem" with "signingAuthority" type from trust store wabbit-networks:
```

Comment:
Same question. I cannot find type of "signingAuthority" in the spec https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md